docs/swarm/how-swarm-mode-works/services.md

```diff
@@ -0,0 +1,101 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/swarm/how-swarm-mode-works/"
```

Comment:
@sfsmithcha yup, that's the plan, and already fixed, see: https://github.com/docker/docker/issues/25118